src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -88,40 +88,39 @@ export class SearchMenu extends React.Component {
         : undefined,
       'type-ahead-options-list': this.state.suggestions,
     });
-  };
 
-  formatTypeaheadSuggestion = suggestion => {
-    const { userInput } = this.state;
-    const remainder = suggestion.replace(userInput, '');
-    return (
-      <>
-        <strong>{userInput}</strong>
-        {remainder}
-      </>
+    const query = suggestion || this.state.userInput;
+
+    const searchUrl = replaceWithStagingDomain(
+      `https://www.va.gov/search/?query=${encodeURIComponent(query)}`,
     );
+
+    window.location.replace(searchUrl);
```

Comment:
I think we should use `window.location.assign` here so that the user's current page is maintained in history (so they can use the back button to go back to their current page after being navigated to the search results)

Reply:
this is a great fix! I'll add it